pytaxize/refactor.py

```diff
@@ -0,0 +1,28 @@
+import requests
+from lxml import etree
+
+def requests_refactor(endpt, payload, request, content=True, format= 'json', **kwargs):
```

Comment:
this `content` parameter is a little confusing. it seems that internally the fxn is switching between json or xml output. change parameter name?